problem_builder/mcq.py

```diff
@@ -51,6 +51,13 @@ class MCQBlock(SubmittingXBlockMixin, QuestionnaireAbstractBlock):
     CATEGORY = 'pb-mcq'
     STUDIO_LABEL = _(u"Multiple Choice Question")
 
+    message = String(
```

Comment:
I moved the message from QuestionnaireAbstractBlock as MCQ and MRQ have different help text